### ISSUE TITLE
Helm upgrade still failing after #3597

### ISSUE THREAD

**Issue by chokdee** [helm/helm#6934]
Output of `helm version`:
version.BuildInfo{Version:"v3.0.0-beta.3", GitCommit:"5cb923eecbe80d1ad76399aee234717c11931d9a", GitTreeState:"clean", GoVersion:"go1.12.9"}

Output of `kubectl version`:
Client Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0+d4cacc0", GitCommit:"d4cacc0", GitTreeState:"clean", BuildDate:"2018-10-10T16:38:01Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}

Server Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0+d4cacc0", GitCommit:"d4cacc0", GitTreeState:"clean", BuildDate:"2019-02-22T07:05:30Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
Openshift

helm ls --all --short
xxx-service-develop

helm upgrade --install --wait --debug --timeout 600s --force --history-max 5 -f ./infrastructure/deployment/xxx-service/values-develop.yaml --set BRANCH_ENVIRONMENT=develop -n pep-xxx-test xxx-service-develop ./xxx-service-16.0.0-203-882dc26.tgz

history.go:52: [debug] getting history for release xxx-service-develop

upgrade.go:79: [debug] preparing upgrade for xxx-service-develop

Error: UPGRADE FAILED: "xxx-service-develop" has no deployed releases

helm.go:81: [debug] "xxx-service-develop" has no deployed releases

helm.sh/helm/pkg/storage.(*Storage).DeployedAll

**Comment by chokdee** [comment]
I have just mentioned that the version Beta 5 have this issue not Beta 3

**Comment by sscholl** [comment]
+1

**Comment by bacongobbler** [comment]
Can you please check the output of `helm history xxx-service-develop`? Also please make sure you're using the latest pre-release (at this time, rc.3). Thanks!

**Comment by bacongobbler** [comment]
possible duplicate of https://github.com/helm/helm/issues/6939?

**Comment by bacongobbler** [comment]
closing as a duplicate of https://github.com/helm/helm/issues/5595.
